cmd/builder/internal/command.go

```diff
@@ -39,6 +39,7 @@ const (
 	distributionOutputPathFlag     = "output-path"
 	distributionGoFlag             = "go"
 	distributionModuleFlag         = "module"
+	distributionBuildTags          = "build-tags"
```

Comment:
Do we need a flag when we have configuration for this? I think we should limit the amount of flags we offer and focus on using the configuration file.

Reply:
@bogdandrutu @TylerHelmuth @jpkrohling If everyone agreed on this, I can remove args input and only support this field in configuration file.

Reply:
Let's start with that in my opinion

Reply:
Can you do the change since others don't respond, we can always add the flag later since it is backwards compatible to add more things.

Reply:
Sorry for delay, I will change this as you said. @bogdandrutu

Reply:
@bogdandrutu I already removed flag definition of buildtags, leaving it only in the config file.

Reply:
Just saw this here. I'm not particularly in favor of flags, but I'm in favor of consistency. Either we clearly mark that we are moving away from flags and won't add this, or we add this to be in line with the other options.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/5946